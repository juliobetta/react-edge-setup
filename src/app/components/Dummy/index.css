.Dummy {
  position: relative;

  div {
    p {
      margin-left: 1rem;
    }
  }
}
