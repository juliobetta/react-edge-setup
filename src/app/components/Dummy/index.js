import React from 'react';
import PropTypes from 'prop-types';
import style from './index.css';

const Dummy = props => (
  <div className={style.Dummy}>
    <h1>React! YAY</h1>
    <div>{props.children}</div>
  </div>
);

Dummy.propTypes = {
  children: PropTypes.oneOfType([PropTypes.element, PropTypes.object])
};

export default Dummy;
