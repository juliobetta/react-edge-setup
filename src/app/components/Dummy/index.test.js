import React from 'react';
import Dummy from './index';

describe('Test React', () => {
  test('Dummy Component', () => {
    const wrapper = mount(<Dummy />);

    expect(wrapper).toMatchSnapshot();
  });
});
