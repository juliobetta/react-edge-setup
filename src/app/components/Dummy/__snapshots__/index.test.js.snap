// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test React Dummy Component 1`] = `
<Dummy>
  <div
    className="Dummy"
  >
    <h1>
      React! YAY
    </h1>
    <div />
  </div>
</Dummy>
`;
